### ISSUE TITLE
No Wyze Cam v3 Pro Support

### ISSUE THREAD

**Issue by anthonystabile** [shauntarves/wyze-sdk#123]
Support for Wyze Cam v3 Pro similar to existing cam support with optional enhancements

    CAMERA_V3_PRO Device model
    Current cameras.py attributes such as:
            "power_switch",
            "temperature",
            "humidity",
            "room_type",
            "comfort_standard_level",
            "supports_temperature_humidity",
            "supports_continuous_record",
            "suppprts_audio_alarm",
            "suppprts_co_alarm",
            "suppprts_motion_alarm",
            "suppprts_smoke_alarm",
            "voltage",
            "battery_charging",
            "power_saving_mode_switch"

    Confirm events model supports new device type.    

In addition to power On/Off support it would be neat to explore these additional use cases:

    Spotlight On/Off
    Siren On/Off
    Smart Focus
    Take Photo
    Restart camera
    Sync Time
    Download images and videos from camera Micro-SD card
    
Shaun, I will share a device for testing.  Thank you for developing the wyze-sdk package.

**Comment by dswhelan** [comment]
I agree with Anthony with regards to API support for additional use cases.  I'm looking for a way to download video segments from a camera's micro-sd card for a given time range.

Thanks for providing and supporting this API.
